similarity/README.md

```diff
@@ -8,43 +8,41 @@ different similarity resources.
 
 ## Download Resource File
 
-Once you first use the specific metrics, the system will automatically download corresponding resource file from CogComp server to `user.home` directory in your local machine.
+When you first use the specific metrics, the system will automatically download corresponding resource file from CogComp server to `user.home` directory in your local machine.
 Notice: Some resource file is very large and it may take a while to download. "paragram" is already included in the `src/main/resources/`.
 
 ## Configure File
 
 The default configure file is `config/configurations.properties.` And see the default config in `SimConfigurator` in `edu.illinois.cs.cogcomp.config` package.
```

Comment:
Why don't we move everything to `SimConfigurator ` and just get rid of `config/configurations.properties`? 

@mssammon  thoughts / preferences on this?